docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  insight:
+    build:
+      context: .
+    network_mode: bridge
+    restart: unless-stopped
+    ports:
+      - 3001:3001
+      - 1989:1989
+    volumes:
+      - ./data/:/bitcoinz/data
```

Comment:
I generally like to keep all directories pertaining to a service together. I wanted to brainstorm with you on this because I noticed you use `/opt/` which is fine. I I generally use symlinks when I want to put data directories elsewhere, like on another drive. I can do so with the docker.sh script, creating a symlink from `/opt/bitcoinz/` into `./data` if you prefer.

Mainly, I don't like having two different directories in the readme for the same data.